### ISSUE TITLE
Replication of preprints

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience#50]
I've been asked if we have a specific policy regarding replication of preprints. There is already a preprint being replicated (https://github.com/ReScience/ReScience-submission/pull/20) and I don't see any problem with preprint replication but I might miss something obvious. Any thought ?

**Comment by labarba** [comment]
I don't object to it, but we should reflect a little: there may be people out there who feel there is some unfairness to the replication study getting formal publication status before the original work. If we are sensitive to these possible bad reactions, we could come up with palliative measures. Does ReScience make any statement recommending that authors of a replication contact the authors of the original work, as a friendly heads-up? Would it be desirable to reach out to authors of a pre-print and ask if they have plans of releasing code and data with the published work?

**Comment by rougier** [comment]
Yes, we could ask re-authors to contact original authors if they're ok with the replication. If so, original authors might even advertise in the paper that the work has been replicated independently. This would strengthen their work in my opinion.

This is also quite compatible with the idea for a  "CoScience" journal where replication is made before and the final article is signed by all authors (yes, I know, a total utopia for science).

**Comment by khinsen** [comment]
We could actually do better than ask authors. We could provide an e-mail template that authors could start from, which would mention the advantages of published replications for the authors of an original study.
